README.md

```diff
@@ -15,8 +15,6 @@ Then:
 git clone git@github.com:NASA-IMPACT/csdap-bulk-download.git
 cd csdap-bulk-download
 uv sync
-uv run pre-commit install
```

Comment:
Unsurprisingly, the notebook stripping stuff didn't work very well and ended up being annoying, so I removed it. **pre-commit** always seems like a good idea before it isn't.